rmw_zenoh_cpp/src/impl/client_impl.cpp

```diff
@@ -11,17 +11,21 @@ std::mutex query_callback_mutex;
 std::atomic<std::int64_t> rmw_client_data_t::sequence_id_counter(0);
 std::atomic<size_t> rmw_client_data_t::client_id_counter(0);
 
+// *INDENT-OFF* because uncrustify can't decide which way to format this
```

Comment:
I am confused by this indentation stuff

Reply:
If you put it on one line, uncrustify tells you to wrap it.
If you wrap it without indentation, uncrustify tells you to indent it.
If you indent it, uncrustify tells you to put it on one line.

So I declared this one of those "they're guidelines, use your common sense" cases, wrapped it so it's easy to read, and told uncrustify to ignore the lines.

Reply:
Aw, but ok